src/server.ts

```diff
@@ -20,7 +20,25 @@ app.disable("x-powered-by");
  **********************************************************************************/
 
 // Common middlewares
-app.use(cors());
+const allowedOrigins = [
+  "https://skylab-frontend.vercel.app/",
+  "http://localhost:3000/",
```

Comment:
```suggestion
  "https://skylab-frontend.vercel.app",
  "http://localhost:3000",
```

i don't think it can have the `/ ` behind